R/susie_ss.R

```diff
@@ -134,14 +137,13 @@ susie_suff_stat = function (bhat, shat, R, n, var_y, XtX, Xty, yty,
     yty = var_y * (n-1)
   }
 
-
   # Check input XtX.
   if (ncol(XtX) != length(Xty))
     stop(paste0("The dimension of XtX (",nrow(XtX)," by ",ncol(XtX),
                 ") does not agree with expected (",length(Xty)," by ",
                 length(Xty),")"))
-  if (!is_symmetric_matrix(XtX))
-    stop("Input XtX or R is not a symmetric matrix")
+  # if (!is_symmetric_matrix(XtX))
```

Comment:
should we really skip this check? and if so, why?

Reply:
Sorry, this was a mistake—I did not mean to comment this out. Should be corrected in commit ad25507.